### PR TITLE
[20.09] Fix mulled-search to not assume conda APIs from vanilla environments

### PR DESCRIPTION
**Pull request by jmchilton**
Fixes https://github.com/galaxyproject/galaxy/issues/10186

### CODE REVIEW

File: lib/galaxy/tool_util/deps/mulled/mulled_search.py [hunk 1]
```diff
@@ -120,6 +120,8 @@ def get_json(self, search_string):
         Function takes search_string variable and returns results from the bioconda channel in JSON format
 
         """
+        if run_command is None:
+            raise Exception("Search with destination conda can only be run from a Conda environment.")
```
Comment (mvdbeek) [inline lib/galaxy/tool_util/deps/mulled/mulled_search.py]: You can also install `galaxy-tool-util[mulled]` (https://github.com/galaxyproject/galaxy/pull/10025)

**Review by mvdbeek — APPROVED** [review]

File: lib/galaxy/tool_util/deps/mulled/mulled_search.py (hunk 1 above)
Comment (jmchilton) [reply lib/galaxy/tool_util/deps/mulled/mulled_search.py]: Oh awesome! I didn't realize pip install conda would work from outside conda - they had pushed back on that years ago. This is cool!

Comment (mvdbeek) [reply lib/galaxy/tool_util/deps/mulled/mulled_search.py]: It's just a small part of the API unfortunately, but it's enough for search to work

Comment (jmchilton) [reply lib/galaxy/tool_util/deps/mulled/mulled_search.py]: Revised the error messages to make this usage more clear.

**Pull request merged**